sherpa/models/basic.py

```diff
@@ -40,14 +41,34 @@
 __all__ = ('Box1D', 'Const1D', 'Cos', 'Delta1D', 'Erf', 'Erfc', 'Exp', 'Exp10',
            'Gauss1D', 'Log', 'Log10', 'LogParabola', 'NormGauss1D', 'Poisson',
            'Polynom1D', 'PowLaw1D', 'Scale1D', 'Sin', 'Sqrt', 'StepHi1D',
-           'StepLo1D', 'Tan', 'Box2D', 'Const2D', 'Delta2D', 'Gauss2D',
-           'SigmaGauss2D',
+           'StepLo1D', 'Tan', 'Voigt',
+           'Box2D', 'Const2D', 'Delta2D', 'Gauss2D', 'SigmaGauss2D',
            'NormGauss2D', 'Polynom2D', 'Scale2D', 'UserModel', 'TableModel',
            'Integrate1D')
 
 DBL_EPSILON = numpy.finfo(numpy.float).eps
 
 
+class Voigt(RegriddableModel1D):
+    """Voigt Profile
+    https://scipython.com/book/chapter-8-scipy/examples/the-voigt-profile/
+    """
+
+    def __init__(self, name='voigt'):
+        self.alpha = Parameter(name, 'alpha', 0.1)
```

Comment:
If I read this correctly, `alpha` and `gamma` are HWHM for the Gaussian and the Lorentian. Instead, I think it would be much better to (a) use more explicit names and (b) be consistent with other models in Sherpa. Looking at the Gauss and Lorentz models, the parameters should use the FWHM as parameters. I suggest `fwhm_g` and `fwhm_l` as names. That way, I can e.g. fit my data with the Gauss model and when I alter find out I need the more complicated Voigt model, then I can just copy the fwhm from the old Gauss as a starting value to this new model. I think that beats both the implementation here and @DougBurke suggestion to use `sigma` as a parameter instead.